cobalt/loader/image/threaded_image_decoder_proxy.cc

```diff
@@ -135,6 +137,11 @@ void ThreadedImageDecoderProxy::Resume(
                  resource_provider));
 }
 
+void ThreadedImageDecoderProxy::Conceal() {
+  base::AutoLock auto_lock(conceal_lock_);
+  is_concealed_ = true;
```

Comment:
Just double checking - this is intentionally one-way, right, i.e. a `ThreadedImageDecoder` will start as not concealed and can then only become concealed and destructed, never resumed again ?

Reply:
It's not entirely clear to me whether or not that's necessary—I added a log to `~ThreadedImageDecoderProxy`, and it is called often, but from reading the code I can't see where it's actually reset (it doesn't look to be during the freeze event, as the destructor does not log between SIGUSR1 and SIGCONT). I would also imagine that if this was broken, rendering would not work on resume because we never actually "finish" rendering an image

Reply:
Talked separately, we're going to leave this as is because it seems to work and will revisit if it seems to cause issues.